Cargo.toml

```diff
@@ -668,6 +668,7 @@ codec = { version = "3.6.12", default-features = false, package = "parity-scale-
 collectives-westend-emulated-chain = { path = "cumulus/parachains/integration-tests/emulated/chains/parachains/collectives/collectives-westend" }
 collectives-westend-runtime = { path = "cumulus/parachains/runtimes/collectives/collectives-westend" }
 color-eyre = { version = "0.6.1", default-features = false }
+const-hex = { version = "1.10.0", default-features = false }
```

Comment:
This is not a new dependency because some of our dependency already depend on it.

I picked up the version already used by some dependencies in the Cargo.lock